### PR TITLE
Added YD60MQ ISO layout

### PR DESCRIPTION
**Pull request by marchein**
## Description

I added support for ISO layout of the YD60MQ pcb. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Added support for ISO layout of the YD60MQ pub

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/yd60mq/keymaps/iso/keymap.c [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    //,: Function Layer
+    [1] = LAYOUT_60_iso(
+        KC_GRV,  KC_F1,   KC_F2,    KC_F3,    KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL,
+        XXXXXXX, KC_BTN1, KC_MS_U,  KC_BTN2,  KC_WH_U, XXXXXXX, XXXXXXX, XXXXXXX, KC_UP,   XXXXXXX, KC_PGUP, KC_HOME, KC_PSCR,
+        KC_TRNS, KC_MS_L, KC_MS_D,  KC_MS_R,  KC_WH_D, RGB_HUI, RGB_SAI, KC_LEFT, KC_DOWN, KC_RGHT, KC_PGDN, KC_END,  XXXXXXX, KC_MNXT,
+        KC_TRNS, RGB_MOD, RGB_RMOD, RGB_VAI,  RGB_VAD, RGB_HUD, RGB_SAD, XXXXXXX, KC_MUTE, KC_VOLD, KC_VOLU, KC_MPLY,          KC_MPRV,
+        KC_TRNS, KC_TRNS, KC_TRNS,                              RGB_TOG,                            RGB_SPI, RGB_SPD, XXXXXXX, KC_TRNS
+	)
+
+};
```
Comment (fauxpark) [inline keyboards/yd60mq/keymaps/iso/keymap.c:28]:
```suggestion
};
```

**Comment by marchein** [comment]
So is there anything left I should do so it can be merged?

**Comment by noroadsleft** [comment]
Applying fauxpark's [suggestion](https://github.com/qmk/qmk_firmware/pull/8815#discussion_r409210034) would probably prompt a second sign-off.

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**